types/swig-email-templates/swig-email-templates-tests.ts

```diff
@@ -1,4 +1,4 @@
-import EmailTemplates = require('swig-email-templates');
+import EmailTemplates from 'swig-email-templates';
```

Comment:
this won't work, once `esModuleInterop` is removed
```suggestion
import EmailTemplates = require('swig-email-templates');
```